view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js

```diff
@@ -164,234 +176,224 @@ define(
-                                        city: city,
-                                        country: country,
-                                        houseNumberOrName: '',
-                                        postalCode: postalCode,
-                                        street: street,
-                                    },
-                                },
-                                onChange: function(state) {
-                                    result.isPlaceOrderAllowed(state.isValid);
                                 }
-                            });
 
-                            try {
-                                result.component = self.checkoutComponent.create(
-                                    result.getBrandCode(), configuration).
-                                    mount(
-                                        '#adyen-alternative-payment-container-' +
-                                        result.getBrandCode());
-                            } catch (err) {
-                                console.log(err);
-                                // The component does not exist yet
-                            }
-                        };
-                        // TODO do the same way as the card payments
-                        result.continueToAdyenBrandCode = function() {
-                            // set payment method to adyen_hpp
-                            var self = this;
-
-                            if (this.validate() &&
-                                additionalValidators.validate()) {
-                                var data = {};
-                                data.method = self.method;
-
-                                var additionalData = {};
-                                additionalData.brand_code = self.brandCode;
-
-                                let stateData;
-                                if ('component' in self) {
-                                    stateData = self.component.data;
-                                } else {
-                                    stateData = {
-                                        paymentMethod: {
-                                            type: self.brandCode
-                                        }
-                                    };
-                                }
+                                /*Use the storedPaymentMethod object and the custom onChange function as the configuration object together*/
+                                var configuration = Object.assign(paymentMethod,
+                                    {
+                                        showPayButton: showPayButton,
+                                        countryCode: country,
+                                        currency: quote.totals().quote_currency_code,
+                                        amount: { // Use this after above removed
+                                            value: 600, // todo get the real amount
```

Comment:
This is not to be replaced in this PR?

Reply:
good one, indeed this needs to be done in this - adding it now

Reply:
Added the changes